openassessment/xblock/static/js/spec/lms/oa_response.js

```diff
@@ -234,35 +234,31 @@ describe("OpenAssessment.ResponseView", function() {
         expect(view.response()[1]).toBe('Test response 2');
     });
 
-    it("updates submit/save buttons and save status when response text changes", function() {
-        // Response is blank --> save/submit buttons disabled
+    it("updates submit button and save status when response text changes", function() {
+        // Response is blank --> submit button disabled
         view.response(['', '']);
         view.handleResponseChanged();
         expect(view.submitEnabled()).toBe(false);
-        expect(view.saveEnabled()).toBe(false);
-        expect(view.saveStatus()).toContain('This response has not been saved.');
+        expect(view.saveStatus()).toContain('This response has not been saved');
 
         // Response is whitespace --> save/submit buttons disabled
         view.response(['               \n      \n      ', ' ']);
         view.handleResponseChanged();
         expect(view.submitEnabled()).toBe(false);
-        expect(view.saveEnabled()).toBe(false);
-        expect(view.saveStatus()).toContain('This response has not been saved.');
+        expect(view.saveStatus()).toContain('This response has not been saved');
 
         // Response is not blank --> submit button enabled
         view.response(['Test response 1', ' ']);
         view.handleResponseChanged();
         expect(view.submitEnabled()).toBe(true);
-        expect(view.saveEnabled()).toBe(true);
-        expect(view.saveStatus()).toContain('This response has not been saved.');
+        expect(view.saveStatus()).toContain('Saving draft');
     });
 
-    it("updates submit/save buttons when response text is optional but file upload is required", function() {
+    it("updates submit button when response text is optional but file upload is required", function() {
         view.textResponse = 'optional';
         view.fileUploadResponse = 'required';
 
         expect(view.submitEnabled()).toBe(false);
-        expect(view.saveEnabled()).toBe(false);
 
```

Comment:
Should this have
expect(view.saveStatus()).toContain('This response has not been saved');
?

Reply:
I hate this checking on strings because it is brittle to any copy changes... but, in lieu of an actual data check, probably?